src/pages/lessons/[slug].tsx

```diff
@@ -1,20 +1,21 @@
 /** @jsx jsx */
 import {jsx} from '@emotion/core'
 import React, {FunctionComponent} from 'react'
+import {GetServerSideProps} from 'next'
 import Link from 'next/link'
 import {useRouter} from 'next/router'
-import EggheadPlayer from 'components/EggheadPlayer'
+import {GraphQLClient} from 'graphql-request'
 import {isEmpty, get} from 'lodash'
 import Markdown from 'react-markdown'
+import {useMachine} from '@xstate/react'
 import useSWR from 'swr'
+import playerMachine from 'components/EggheadPlayer/machine'
+import EggheadPlayer from 'components/EggheadPlayer'
+import PlayerControls from 'components/PlayerControls'
+import Metadata from 'components/Metadata'
 import {loadLesson} from 'lib/lessons'
-import {GraphQLClient} from 'graphql-request'
 import {useViewer} from 'context/viewer-context'
-import {GetServerSideProps} from 'next'
-import {LessonResource} from 'types'
-import {useMachine} from '@xstate/react'
-import playerMachine from 'components/EggheadPlayer/machine'
-import Eggo from '../../../public/images/eggo.svg'
+import * as Types from 'types'
```

Comment:
As a suggestion: what if we import types in this way `import * as Types from 'types'` and use it as `Types.Lesson` for example? Just to avoid intersection between `Lesson` type and `Lesson` component? Just a matter of naming convention

Reply:
I prefer to import them Individually and keep the code below less  wrbose and having a few extra lines of imports